### DISCUSSION TITLE
Backup email should be able to be already registered as another account's primary email address.

### DISCUSSION THREAD

**Discussion by RadxaYuntian** [community/community#129553] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

GitHub is now asking me to add another email address to my work account in case if I lose access to my work email. I guess not the worst of suggestion, so I put my personal email in, and got notified that the email was already registered.

If that email is only for password recovery, I don't see why it can't be registered with another account. Just disable log in on it. Otherwise for each GitHub account I'd want to have 2 unique email accounts.

Now just waiting for some email providers asking you to connect 2 GitHub accounts in case you lose access to your primary GitHub account.

**Comment by knordstrom-muon** [comment]
Yep this makes the "We recommend verifying at least one more email address" warning useless for work accounts -- I'm not going to create a new personal email account for this.

**Comment by neil-m** [comment]
Glad i'm not the only one confused by this, i don't know what to do with my work account. Every coworker and every personal email i have is already registered in some way to github. So i don't have any emails left to use, why can't i reuse my personal one?

**Comment by JamesSteiningerSciton** [comment]
Agreed, this is a really unhelpful solution to not be able to use an email tied to an existing account. There's no way I'm making a backup email uniquely for GitHub usage. I'm more likely to lose access to that email than my personal or work email.
